brands/healthcare/clinic.json

```diff
@@ -0,0 +1,13 @@
+{
+  "healthcare/clinic|ZoomCare": {
```

Comment:
(Note that a `matchNames` of `zoom+care` is unnecessary because plus signs are removed in fuzzy matching.)

Reply:
Yeah I figured that would just magically happen here